composer.json

```diff
@@ -4,15 +4,17 @@
     "keywords": ["event-loop", "asynchronous"],
     "license": "MIT",
     "require": {
-        "php": ">=5.4.0"
+        "php": ">=5.4.0",
+        "evenement/evenement": "^3.0||^2.0"
```

Comment:
I know this dependency is used throughout the ecosystem, but is it really worth it or should we try to keep this component stand-alone?